druntime/src/core/stdc/stdatomic.d

```diff
@@ -169,6 +169,7 @@ void atomic_flag_clear_explicit_impl()(atomic_flag* obj, memory_order order)
             break;
 
         case memory_order.memory_order_acquire:
+        case memory_order.memory_order_acq_rel:
```

Comment:
Does C similarly translate invalid orders to valid ones?

Reply:
Okay, so if they do compile in C, albeit with a warning, I guess it's fine to accept them here in stdatomic too. - I have no idea about the orders, so cannot comment on whether the mapping chosen here is fine, but I have faith in Iain. :)

Reply:
The orders are invalid, so the mappings aren't meant to be "fine", it's just fallback to most expensive.